src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -118,10 +143,13 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
       documentUpload.append('files', this.selectedFile, this.selectedFile.name);
       documentUpload.append('classification', 'PUBLIC');
       this.documentManagement.uploadFile(documentUpload).subscribe(result => {
-        if (!this.uploadedDocument ) {
-          this.createDocumentGroup();
+        if (!this.uploadedDocument) {
+            this.uploadedDocument = this.registerControl(new FormGroup({
+              document_url: new FormControl(WriteDocumentFieldComponent.DOCUMENT_URL),
```

Comment:
can u reuse `setDocumentFormValues` method?
Furthermore, why we setting the values to `WriteDocumentFieldComponent.DOCUMENT_URL`, `WriteDocumentFieldComponent.DOCUMENT_BINARY_URL` and `WriteDocumentFieldComponent.DOCUMENT_FILENAME`?